nixos/modules/security/ca.nix

```diff
@@ -8,12 +8,9 @@ let
 
   cacertPackage = pkgs.cacert.override {
     blacklist = cfg.caCertificateBlacklist;
+    extraCertificates = cfg.certificateFiles ++ cfg.certificates;
```

Comment:
So this means we will require the full NSS source on every single machine that sets  a custom certificate? How much is it these days?

Reply:
The answer is about 80MB gzip compressed. Could we introduce an intermediary step that extracts the files that we need for this and only use that as src here? That would allow downloading a miminal set of files (compressd) from the binary cache vs having to obtain the full source archive.

Reply:
We could probably change this to take the output of the cacert package and then filter/add to _that_ instead; then if you block a CA it has the same cost as adding a new one.

Reply:
I did the simpler thing you suggested because that's less effort ;)